### ISSUE TITLE
[BUG] WK Loss only meaningful when prediction is extremely close to actual label

### ISSUE THREAD

**Issue by agaldran** [ayrna/dlordinal#112]
### Describe the bug

Hello,

I am not super-sure of this being a bug or a mistake of mine, but I am trying to use the QWK loss and I always get a value of 1 (unless I pass a perfect prediction). In fact, I need to subtract a tiny epsilon from the perfect prediction in order to see some loss variation. Is this intended behavior? Thanks for the library!

### Steps/Code to reproduce the bug

```
from dlordinal.losses import WKLoss

loss_fn = WKLoss(num_classes=3, penalization_type='quadratic', use_logits=False)

# Example 1: 
logits1 = torch.tensor([[0.90, 0.05, 0.05]])
target1 = torch.tensor([0])
loss1 = loss_fn(logits1, target1)
print("Loss1:", loss1.item()) # Loss1: 1.0

# Example 2: 
logits2 = torch.tensor([[0.95, 0.025, 0.025]])
target2 = torch.tensor([0])
loss2 = loss_fn(logits2, target2)
print("Loss2:", loss2.item()) # Loss2: 0.9999998807907104

# Example 3: 
logits3 = torch.tensor([[0.9999, 0.00005, 0.00005]])
target3 = torch.tensor([0])
loss3 = loss_fn(logits3, target3)
print("Loss3:", loss3.item()) # Loss3: 0.9689982533454895

# Example 4: 
logits4 = torch.tensor([[1., 0., 0.]])
target4 = torch.tensor([0])
loss4 = loss_fn(logits4, target4)
print("Loss4:", loss4.item()) # Loss4: 0.0
```

### Expected results

I expected a more varied range of loss values

### Actual results

Range of loss values is very small.

**Comment by agaldran** [comment]
Hi Victor, upon further reflexion, I realize that one should not be able to meaningfully compute a QWK loss over a single sample but rather needs a set of predictions and labels (and the larger the better), same as you cannot do that with the AUC, so maybe the source of the behaviour above is just that.
